### ISSUE TITLE
Comments and questions about the randomizer

### ISSUE THREAD

**Issue by dombarrieau** [WordofWind/Dark-Cloud-2-Chest-Randomizer#2]
Hey, while I don't have a particular issue per say, I have some questions and comments as I just finished a run through the game with the randomizer. First of all, I loved it, definitely gave a new feel to the game I've played so many times already.

Question 1. Is Witch's Parfait missing from the item pool? I realize I may have just been screwed by RNG, but I opened many Fruit of Edens (to the point where both characters had maxed HP at 255, and I couldn't get rid of the extra ones), and Potato Pies, but nothing to raise Monica's Defense.

Question 2. I received three Monster Drop items from chests (what you get when you level up monsters to certain points), but all three gave 0 stat increases when spectrumized, so I simply sold them for money. Is this expected? Since each you obtain legit have different value, the base item you get from randomized chests is blank.

Other comment. Clown chests felt useless most of the time, since both items were bad, maybe one of them could have a higher value item?

Overall loved it, Would be interested to see what else you work on over time for DC games! <3

**Comment by WordofWind** [comment]
Hey,

1. Witch Parfait should definitely be part of the item pool. Maybe you just got really unlucky to not find any?
2. Yeah, I was aware that the Monster Drops don't give stats, but since I noticed they can be sold for some amount of money, I decided to keep them.

As for the clowns, the both items are similar to a random chest loot, so its likely that they both are common stuff. I suppose I could have made one of them to be guaranteed higher value, but maybe I just didn't "bother" with that I guess.
